maltoolbox/language/languagegraph.py

```diff
@@ -888,74 +1390,149 @@ def _generate_graph(self) -> None:
                     type = attack_step_attribs['type'],
                     asset = asset,
                     ttc = attack_step_attribs['ttc'],
+                    overrides = attack_step_attribs['reaches']['overrides'] \
+                        if attack_step_attribs['reaches'] else False,
                     children = {},
                     parents = {},
-                    description = attack_step_attribs['meta']
+                    description = attack_step_attribs['meta'],
+                    tags = set(attack_step_attribs['tags'])
                 )
                 attack_step_node.attributes = attack_step_attribs
```

Comment:
I think this should be a private property as it is used only here to help with the separate loops and is not meant as public interface of the attack step node.